src/Core/Grid/Row/RowCollection.php

```diff
@@ -48,4 +48,16 @@ public function getRows()
     {
         return $this->items;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function map(callable $modifier)
+    {
+        foreach ($this->items as $key => $row) {
+            $this->items[$key] = $modifier($row);
```

Comment:
It's maybe better to use `call_user_func`, a string is callable but not working with `$modifier()`.
Not working in 5.6 but working in 7+ http://sandbox.onlinephpfunctions.com/code/fd2eefda032f0c74a26662a68156f80b4710f962